### PR TITLE
[Tests-Only] Add tests for valid and invalid file/folder name

### PR DESCRIPTION
**Pull request by swoichha**
## Description
There are test scenarios for renaming files/folders to "invalid" file names. Some of these names are invalid for oC10 but are allowed when the server back-end is OCIS. So, this PR add separate these types of tests to run successfully on oC10 and ocis accordingly by using tag `@notToImplementOnOCIS`  and  `@skipOnOC10`

Also adjusted expected failures files as needed.

## Related Issue
- Fixes https://github.com/owncloud/web/issues/4587

## How Has This Been Tested?
:robot: 

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Technical debt
- [x] Tests

## Checklist:

- [ ] Code changes
- [ ] Unit tests added
- [x] Acceptance tests added
- [ ] Documentation ticket raised: <link> 

## Open tasks:

- [ ] ...

### CODE REVIEW

File: tests/acceptance/features/webUIRenameFiles/renameFiles.feature [hunk 1]
```diff
@@ -102,6 +102,7 @@ Feature: rename files
     Then file "loremy.tad" should be listed on the webUI
 
 
+  @notToImplementOnOCIS
```
Comment (phil-davis) [inline tests/acceptance/features/webUIRenameFiles/renameFiles.feature:105]: No need to add a line here - there are already 2 blank lines. One of then is waiting for tags.

File: tests/acceptance/features/webUIRenameFiles/renameFiles.feature [hunk 2]
```diff
@@ -156,11 +167,17 @@ Feature: rename files
     And file 'data.zip' should be listed on the webUI
 
 
+  @notToImplementOnOCIS
```
Comment (phil-davis) [inline tests/acceptance/features/webUIRenameFiles/renameFiles.feature:170]: Same here - no need for another blank line

File: tests/acceptance/features/webUIRenameFolders/renameFolders.feature [hunk 3]
```diff
@@ -94,6 +94,7 @@ Feature: rename folders
       | another normal folder |
 
 
+  @notToImplementOnOCIS
```
Comment (phil-davis) [inline tests/acceptance/features/webUIRenameFolders/renameFolders.feature:97]: same here - extra line not needed

File: tests/acceptance/features/webUIRenameFolders/renameFolders.feature [hunk 4]
```diff
@@ -121,11 +132,16 @@ Feature: rename folders
     Then the error message 'The name cannot be equal to "."' should be displayed on the webUI dialog prompt
 
 
+  @notToImplementOnOCIS
```
Comment (phil-davis) [inline tests/acceptance/features/webUIRenameFolders/renameFolders.feature:135]: and here - new line not needed

**Review by dpakach — COMMENTED** [review]
LGTM :+1:

**Review by phil-davis — APPROVED** [review]

**Pull request merged**